types/stripe-js/stripe.d.ts

```diff
@@ -38,6 +38,14 @@ export interface Stripe {
     options: RedirectToCheckoutOptions
   ): Promise<never | {error: StripeError}>;
 
+  /**
+   * Use stripe.createRadarSession to create a [Radar Session](https://stripe.com/docs/radar/radar-session) in your checkout flow or when saving card details.
+   * A Radar Session is a snapshot of the browser metadata and device details that helps Radar make more accurate predictions on your payments.
+   * This metadata includes information like IP address, browser, screen or device information, and other device characteristics.
+   * By using Radar Sessions, you can capture critical fraud information without tokenizing on Stripe.
+   */
+  createRadarSession(): Promise<RadarSessionPayload>;
```

Comment:
It looks like `createRadarSession` is under 'payment intents' in the [docs](https://stripe.com/docs/js/payment_intents/create_radar_session) - could this be placed under that section (right below here)? Maybe directly below `verifyMicrodepositsForPayment` since that's the order in the docs?